Makefile

```diff
@@ -97,6 +97,7 @@ bin/$(ARCH)/$(BIN): build-dirs
 	    -v $$(pwd)/bin/$(ARCH):/go/bin/linux_$(ARCH)                       \
 	    -v $$(pwd)/.go/std/$(ARCH):/usr/local/go/pkg/linux_$(ARCH)_static  \
 	    -w /go/src/$(PKG)                                                  \
+            -e GOCACHE=off                                                     \
```

Comment:
Is this to avoid permission error like below?
```
go: disabling cache (/.cache/go-build) due to initialization failure: mkdir /.cache: permission denied
```

Reply:
It is. If I recall correctly that error was causing the build to fail with Go 1.11.